src/commands/app/add/event.js

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+const BaseCommand = require('../../../BaseCommand')
+const yeoman = require('yeoman-environment')
+const debug = require('debug')('aio-cli-plugin-app:init')
+const { flags } = require('@oclif/command')
+
+const config = require('@adobe/aio-lib-core-config')
+
+class AddEventCommand extends BaseCommand {
+  async run () {
+    const { args, flags } = this.parse(AddEventCommand)
+
+    debug(`adding component ${args.component} to the project, using flags: `, flags)
+
+    const services = (config.get('services') || []).map(s => s.code).join(',')
```

Comment:
please remove the services argument, it's not used by the add-events generator..